testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kerberos-kms-hdfs-no-impersonation-with-credential-cache/iceberg.properties

```diff
@@ -0,0 +1,16 @@
+connector.name=iceberg
+hive.metastore.uri=thrift://hadoop-master:9083
+
+hive.metastore.authentication.type=KERBEROS
+hive.metastore.service.principal=hive/_HOST@LABS.TERADATA.COM
+hive.metastore.client.principal=presto-server/_HOST@LABS.TERADATA.COM
+hive.metastore.client.credential-cache.location=/etc/trino/conf/presto-server-krbcc
+
+hive.hdfs.authentication.type=KERBEROS
+hive.hdfs.impersonation.enabled=false
+hive.hdfs.trino.principal=presto-server/_HOST@LABS.TERADATA.COM
```

Comment:
Realm can be omitted actually as it has a default in krb5.conf:

```
[libdefaults]
 default_realm = LABS.TERADATA.COM
```

Reply:
But the `_HOST` replacement (which is also available in Hadoop) supports only for pattern of Principal with Realm. So for maintaining consistency it with hadoop's configuration it would be better if we could specify it as a part of config. Ref: https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SecureMode.html#Kerberos_principals_for_Hadoop_Daemons